.github/workflows/tests.yml

```diff
@@ -80,14 +83,11 @@ jobs:
       - name: Run PennyLane-Lightning unit tests
         run: |
           cd main/
-          pytest tests/ --cov=pennylane_lightning --cov-report term-missing --cov-report=xml:../coverage.xml
+          pytest tests/ $COVERAGE_FLAGS
+          pl-device-test --device lightning.qubit --skip-ops --shots=20000 $COVERAGE_FLAGS
+          pl-device-test --device lightning.qubit --shots=None --skip-ops $COVERAGE_FLAGS
```

Comment:
:thinking: I'm not sure if I broke codecov, or if it's just because this is a branch of `7527_unitary_gate`
![image](https://user-images.githubusercontent.com/49409390/128565936-8e272460-ea1a-46e0-98f0-6072ad6db7a0.png)